glslc/README.asciidoc

```diff
@@ -349,11 +350,27 @@ Use HLSL packing rules instead of GLSL rules when determining offsets of
 members of blocks.  This option is always on when compiling for HLSL.
 
 [[option-fentry-point]]
-==== `-fentry-point=`
+==== `-fentry-point=<name>`
 
 `-fentry-point=<name>` lets you specify the entry point name.  This is only
 significant for HLSL compilation.  The default is "main".
 
+[[option-fauto-map-locations]]
+==== `-fauto-map-locations`
+
+Option `-fauto-map-locations` directs the compiler to automatically assign
+location numbers to user-defined stage input and output variables.
+
+Client APIs normally require adjacent stages to agree on their I/O interface.
+The compiler only sees one stage at a time, so it is strongly recommended that
+you avoid relying on this option to assign locations.
+
+Instead, an explicit binding number should be specified in the shader source by
+using a `location` layout qualifier.  For example:
```

Comment:
It's unclear to me whether this feature is specifically for GLSL (which the example is) or both GLSL and HLSL. For HLSL, I think we should promote `[[vk::location(X)]]` as the suggested way since it is supported by both compilers.